docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-content-manager-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='using-standalone-perspectives-content-manager-proc']
+= Using standalone content manager perspective
+
+By using the standalone content manager perspective in your application, you can create and edit your application's content and its navigation menus.
+
+.Procedure
+. Login to {CENTRAL}.
+. In a web browser, enter the following link in its address bar:
```

Comment:
Might be better to reword as follows: 
Enter the following URL in a web browser:

Reply:
Changed as suggested.